### DISCUSSION TITLE
Неправильно показываются координаты

### DISCUSSION THREAD

**Discussion by ValievE** [yandex-maps-unofficial/vue-yandex-maps#300] [Q&A (2.x)] [answered]
Добрый день. Задам еще один вопросик) Карта отобразилась, взял координаты нужного здания из Яндекс.Карт и вставил в код, но на сайте карта отобразилась на примерно 10 км правее и ниже. С чем это может быть связано и как починить?

UPD: удалять не стану, но если у кого-то такая же проблема - решение достаточно простое, яндекс координаты выдает таким видом - [X,Y], а в коде надо написать координаты видом [Y,X] и все встало как надо))

**Comment by daniluk4000** [comment, marked as answer]
Решено автором
